docs/usage.md

```diff
@@ -189,7 +190,7 @@ Linear model function to calculate the contrasts (TXT). Variable names should be
 
 Project summary as downloaded from the portal: User database portlet, Projects tab, select your project and "Download Project Information". Please check first that this information is correct in the project Browser.
 
-### `--versions`
+### `--software_versions`
 
 Path to the `Software_versions.csv` file generated by the RNAseq pipeline.
```

Comment:
The rnaseq-pipeline does not provide a `.csv` but a `.yml` file. Can you provide instructions on how to generate one or show an example excerpt for users so that they don't have to inspect the raw.githubusercontent testdata?
Otherwise you could consider accepting a .yml file as well or instead

Reply:
``
```suggestion
Path to the `Software_versions.csv` file generated by the nf-core/rnaseq pipeline. 

The structure should look like this:
nf-core/rnaseq	v1.4dev
Nextflow	v19.01.0
FastQC	v0.11.8
Cutadapt	v2.1
Trim Galore!	v0.5.0
STAR	vSTAR_2.6.1d
```

Reply:
Maybe you should also change the file extension to `tsv` because even github complains that the `csv` file is not comma-separated, see [here](https://github.com/qbic-pipelines/rnadeseq/blob/master/testdata/software_versions.csv) and users will be less confused why a comma-separated file is not accepted.